### PR TITLE
Filter grid issue

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were errors, for info, please see...

**Comment by admin-vizcentric** [comment]
There were errors, for info, please see...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/content/scss/partials/_login-page.scss [hunk 1]
```diff
@@ -43,6 +43,13 @@
                     margin-bottom: 2px;
                 }
             }
+            .SS0{
+                color: #19427D;
```
Comment (johibkhan2) [inline src/main/webapp/content/scss/partials/_login-page.scss:47]: Create scss variable for colours

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**